### PR TITLE
Added buttons to contact user

### PR DESCRIPTION
**Pull request by tenbradley**

### CODE REVIEW

File: user-profile-card.js [hunk 1]
```diff
@@ -118,10 +120,11 @@ class UserProfileCard extends LocalizeMixin(LitElement) {
 		}
 		.d2l-profile-card-media {
 			display: flex;
-			justify-content: space-between;
+			align-items: center;
 		}
 		::slotted([slot=website]) {
-			margin-top: 11px;
+			margin-left: 40px;
+
```
Comment (lloydwaddell) [inline user-profile-card.js:127]: extra newline can be removed

**Review by lloydwaddell — APPROVED** [review]

File: user-profile-card.js [hunk 2]
```diff
@@ -238,10 +245,28 @@ class UserProfileCard extends LocalizeMixin(LitElement) {
 					<slot name="awards-icons"></slot>
 				</div>
 				<div class="d2l-labs-profile-card-contact">
-					<slot name="contact"></slot>
+					<div class="d2l-profile-card-contact-info">
+						<div>
+							<d2l-button-subtle id="email" text="Email" icon="tier1:email" @click="${this._onEmailClick}"></d2l-button-subtle>
+							<d2l-button-subtle id="message" text="Instant Message" icon="tier1:add-message" @click="${this._onMessageClick}"></d2l-button-subtle>
+						</div>
+						${ this.progressViewable ? html`<d2l-button-subtle id="progress" text="User Progress" icon="tier1:user-progress" @click="${this._onProgressClick}"></d2l-button-subtle>` : html`` }
```
Comment (m6jones) [inline user-profile-card.js:253]: Seeing the code this is good! We can then emit an event saying progress was hit and the user of the component can trigger off that.

File: user-profile-card.js [hunk 3]
```diff
@@ -238,10 +245,28 @@ class UserProfileCard extends LocalizeMixin(LitElement) {
...
+		console.log('email');
+	}
+
+	_onMessageClick() {
+		console.log('message');
+	}
+
+	_onProgressClick() {
+		console.log('progress');
+	}
```
Comment (m6jones) [inline user-profile-card.js:270]: These just need to emit events and we are done!

File: user-profile-card.js [hunk 4]
```diff
@@ -238,10 +245,28 @@ class UserProfileCard extends LocalizeMixin(LitElement) {
...
+							<d2l-button-subtle id="message" text="Instant Message" icon="tier1:add-message" @click="${this._openMessage}"></d2l-button-subtle>
+						</div>
+						${ this.progressViewable ? html`<d2l-button-subtle id="progress" text="User Progress" icon="tier1:user-progress" @click="${this._userProgress}"></d2l-button-subtle>` : html`` }
+					</div>
 				</div>
 			</div>
 		`;
 	}
+
+	_openEmail() {
```
[the remark this answers is not in the document]
Comment (m6jones) [reply user-profile-card.js]: Yes! I'll grab a link for you to read that goes over this.

File: user-profile-card.js [hunk 5]
```diff
@@ -59,7 +61,7 @@ class UserProfileCard extends LocalizeMixin(LitElement) {
 				grid-column: start / end;
 				grid-row: contact-start / contact-end;
 				border-top: 1px solid var(--d2l-color-mica);
-				background-color: var(#f9fbff)
+				background-color: #f9fbff
```
Comment (m6jones) [inline user-profile-card.js:64]: Use d2l color. Look above.

File: user-profile-card.js [hunk 6]
```diff
@@ -168,13 +171,16 @@ class UserProfileCard extends LocalizeMixin(LitElement) {
 		.d2l-labs-profile-card-contact {
 			grid-column: start / end;
 			grid-row: media-start / media-end;
-			padding: 22px 26px;
+			padding: 22px 22px;
```
Comment (m6jones) [inline user-profile-card.js:174]:
```suggestion
			padding: 22px;
```

**Pull request merged**